frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.module.css

```diff
@@ -88,6 +88,13 @@
   position: absolute;
   width: 23px;
   height: 16px;
+}
+
+.handleCardinalityHighlighted {
+  color: var(--node-layout);
+}
+
+.handleCardinalityDefault {
```

Comment:
This class can be deleted!
This color could be implemented in `handleCardinality` and overridden when highlighted to make it more CSS-like.

Reply:
That makes sense!
c5838daabfed87e75480d5534ff2a5c0d62dd552

Reply:
I want to check in a preview environment that it will cascade properly. loading...

Reply:
https://github.com/user-attachments/assets/9cd587d2-4c0e-4da7-aa42-44d09db7bfca

Nice work 🚀 
However, the following adjustments should be made! Of course, you can use a different PR.

- No animation on hover, only cardinality is highlighted first, please add animation.
- Fix the edges that are still visible behind the cardinality.

<img width="608" alt="スクリーンショット_2024-12-12_15_27_56" src="https://github.com/user-attachments/assets/a7fdefa9-b861-4f3f-a877-d6aecc4fa12e" />

It would be best if Edge was connected directly to Cardinality...

Reply:
Yeah, it was important.
Is this good?
6056d0203811a3b251b5a70d13f0a3ca6a3d91ca

Reply:
https://github.com/user-attachments/assets/5b8b6d10-9e66-4478-b795-d50a60fdf279

Reply:
COOOOOOOOOOL!!!!!!!!!!!!!!!!!!!!!

Reply:
This PR is blocking other PRs. So let me fix this another one 👍

Reply:
Send the text at the same time😅
Thank you confirmation!